src/components/Package/Package.tsx

```diff
@@ -45,7 +40,7 @@ export interface PackageInterface {
   name: string;
   version: string;
   time?: number | string;
-  author: Author;
+  author: Required<PackageMetaInterface['latest']>['author'];
```

Comment:
Hmmm I would rather use the Author interface + string here.

Reply:
I will resolve this one, above is already explained.

Reply:
I used this gross type to avoid renaming the `Author` type import to fix the conflict with the `Author` styled component because I couldn't think of something to rename it to. I went with `PackageAuthor` - not ideal but maybe a bit better than this